doc/man3/CMS_EnvelopedData_create.pod

```diff
@@ -24,6 +27,11 @@ L<CMS_add0_recipient_key(3)>.
 The B<CMS_ContentInfo> structure needs to be finalized using L<CMS_final(3)>
 and then freed using L<CMS_ContentInfo_free(3)>.
 
+
+CMS_EnvelopedData_create_with_libctx() is similar to CMS_EnvelopedData_create()
```

Comment:
I've tended to put the `with_libctx` version first in the docs and then describe how the non-libctx version is different.

Reply:
ok